### PR TITLE
Answer: "Why is ETH switching to Proof of Stake (Pos)?"

### PR DESCRIPTION
**Pull request by Myu-Unix**
Hello /u/InsideTheSimulation/ , /u/Chapo_Rouge reporting for duty :)

Did I did this right ? I want to answer to question with weight "2.0" ? I am a bit unsure.

Cheers !

### CODE REVIEW

**Comment by InsideTheSim** [comment]
@Myu-Unix Hey there! Looking good! Thanks for the contribution. Since this is an existing question in the repo please make your edits to the exsting file here as opposed to creating a new one.

https://github.com/InsideTheSim/ethmerge.com-content/blob/main/faqs/why-is-eth-switching-to-pos.md

👍

File: faqs/why-is-eth-switching-to-pos.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 ---
 title: Why is ETH switching to Proof of Stake (Pos)?
-weight: 2.0
+weight: 2.1
```
Comment (InsideTheSim) [inline faqs/why-is-eth-switching-to-pos.md:3]: You can leave the weight at 2.0 since we're not adding an additional question here.

**Comment by Myu-Unix** [comment]
Alright, done on the existing file ! :)

**Review by InsideTheSim — COMMENTED** [review]
Thank you! This is the right place now. 😄

I like your content - thanks for working it up. The only concern I have (and maybe this is just me) is that Binance has a bad rep with the Ethereum community right now due to their BSC chain. Are there any other sources you'd trust to list for explanations of PoW vs PoS?

**Comment by Myu-Unix** [comment]
> Thank you! This is the right place now. smile
> 
> I like your content - thanks for working it up. The only concern I have (and maybe this is just me) is that Binance has a bad rep with the Ethereum community right now due to their BSC chain. Are there any other sources you'd trust to list for explanations of PoW vs PoS?

Thanks for the kind words and thank you for setting the various websites up ! :)

So there's other sources for PoW vs PoS on Youtube indeed but I find that they are a fair bit less professionally made and usually much longer than 2-3 mins or bundled together in a "PoW vs PoS video" which is a bit less fitting for my text. I share the same feeling about BSC too but I got to give to Binance Academy that these videos are well crafted indeed.

**Comment by InsideTheSim** [comment]
Ok, good enough for me! If someone else has better sources they can submit them. 💪

Thank you for your contribution! Merging.

**Pull request merged**